src/ExtraClass.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace MaximeRainville\SilverstripeReact;
+
+/**
+ * Apply this trait to ViewableData classes that can have extra classes applied to them.
+ */
+trait ExtraClass
```

Comment:
This trait probably should be in Silverstripe Framework. I'll do a PR to add it later.